pkg/deploy/generator/const.go

```diff
@@ -0,0 +1,24 @@
+package generator
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+// Deployment constants
+const (
+	KeyVaultTagName          = "vault"
+	ClustersKeyVaultTagValue = "clusters"
+	ServiceKeyVaultTagValue  = "service"
+	kvClusterSuffix          = "-cls"
+	kvServiceSuffix          = "-svc"
+)
```

Comment:
I am tempted to leave these where they were.  I don't see the need for the churn of moving them.  Happy to add the new constants in this package though.

Reply:
Circular imports if we move back to `deploy` pkg as they are used by generators itself.